### DISCUSSION TITLE
Approved for GitHub Education, but Benefits Not Received Yet.

### DISCUSSION THREAD

**Discussion by alphacoder7817** [community/community#138392] [GitHub Education]
### Hi everyone,

Hi GitHub Education Team,

I **applied for GitHub Education and got approved** with a message saying that 

>my benefits would be available soon.
![Screenshot 2024-09-11 200431](https://github.com/user-attachments/assets/162f93d5-0769-480d-8a01-943f2a4fcbbd)

However, it's been **5 days,** and I haven’t received any benefits yet.

Could someone assist me with this?

Thanks!

**Comment by pattherogue** [comment]
I am going through the same issue myself. Could someone help me with this as well.

**Comment by davevad93** [comment]
Hi @alphacoder7817 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @alphacoder7817 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
